### PR TITLE
chore: align thirdparty docs, seq diagrams and oa3 files

### PR DESCRIPTION
**Pull request by kleyow**
@lewisdaly getting this error when trying to build the puml svg's using `_build_plantuml_all.sh`

```
[kleyow@lavi assets] (chore/align) ./_build_plantuml_all.sh
bdd449be59b79cbe196549ce60802a4da97f02932eb23c4952c7893813a1a48c
rendering .puml -> .svg for diagram diagram: /home/kleyow/mojaloop/documentation/docs/api/thirdparty/assets/diagrams/transfer/1-2-3-signed-authorization-fido.puml
http request failed to render puml with error connect ECONNREFUSED 127.0.0.1:80
```

### CODE REVIEW

File: website/versioned_docs/v1.0.1/api/thirdparty/data-models.md [hunk 1]
```diff
@@ -879,7 +879,6 @@ The AuthenticationResponse enumeration describes the result of authenticating ve
 | Name | Description |
 | ---  | ----------- |
 | VERIFIED | The challenge was correctly signed. |
-| REJECTED | The challenge was not correctly signed. |
```
Comment (kleyow) [inline website/versioned_docs/v1.0.1/api/thirdparty/data-models.md:882]: Sorry about the diff here. My editor trims trialing white space. This is the only real edit.

**Comment by lewisdaly** [comment]
> @lewisdaly getting this error when trying to build the puml svg's using `_build_plantuml_all.sh`
> 
> ```
> [kleyow@lavi assets] (chore/align) ./_build_plantuml_all.sh
> bdd449be59b79cbe196549ce60802a4da97f02932eb23c4952c7893813a1a48c
> rendering .puml -> .svg for diagram diagram: /home/kleyow/mojaloop/documentation/docs/api/thirdparty/assets/diagrams/transfer/1-2-3-signed-authorization-fido.puml
> http request failed to render puml with error connect ECONNREFUSED 127.0.0.1:80
> ```

Hmm. It runs in a docker container, so maybe look at your docker logs to see what's going on. You might need to increase the sleep time to wait for the container to be running. It's not an elegant solution but it should work.

**Comment by lewisdaly** [comment]
Ah wait, we're on the documentation repo! Yeah I started setting up git hooks, but haven't finished yet. I'm not sure why but it looks like those hooks stuck even though I originally applied them to a different branch. The mysteries of `.git` I suppose.

**Review by lewisdaly — APPROVED** [review]

**Comment by lewisdaly** [comment]
Maybe disable the build step for now if you can't fix it easily - I'm working on that in this PR: https://github.com/mojaloop/documentation/pull/349

**Comment by kleyow** [comment]
> Hmm. It runs in a docker container, so maybe look at your docker logs to see what's going on. You might need to increase the sleep time to wait for the container to be running. It's not an elegant solution but it should work.

@lewisdaly Still can get the puml diagram's to build, don't think it's the docker image being slow to boot up.

> Maybe disable the build step for now if you can't fix it easily - I'm working on that in this PR: https://github.com/mojaloop/documentation/pull/349

Sure I can disable it.

One last question. Why are docs/ and website/versioned_docs similar? Which one should I be changing? Both?

**Comment by elnyry-sam-k** [comment]
@kleyow - would like to follow this up.. Lets discuss what's needed to get this over the line.. Thanks!

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**